mathml/relations/css-styling/dynamic-dir-1-ref.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Test dynamically changing dir attribute</title>
+  <link rel="help" href="https://mathml-refresh.github.io/mathml-core/#cssproperties">
```

Comment:
From https://web-platform-tests.org/writing-tests/reftests.html#example-reftest it seems we remove all metadata from the ref.

Reply:
ugh, whoops. thanks.